bin/check-lv-usage.rb

```diff
@@ -72,7 +72,7 @@ class CheckLVUsage < Sensu::Plugin::Check::CLI
          default: 95
   # Setup variables
   #
-  def initialize
+  def initialize(argv = ARGV)
     super
```

Comment:
This shouldn't be needed. `Sensu::Plugin::CLI` (the base class which uses `argv`) already has the default of `argv=ARGV` -- you're not using this variable yourself so I think this just looks strange.